H_ecto_models.md

```diff
@@ -582,7 +585,7 @@ defmodule HelloPhoenix.Video do
 . . .
 end
 ```
-Note that we don't declare the field `user_id` in the video model, but, as a required (or optional) field, we do declare the `user_id` field.
+Note that we don't declare the field `user_id` in the video model schema. We add it to required (or optional) field list instead.
```

Comment:
I think we could use a definite article here as well:

We add it to the required . . .